### PR TITLE
Remove support for Python 3.7 to align with keras-nightly

### PR DESCRIPTION
**Pull request by sampathweb**
Keras Nightly & TensorFlow Nightly have dropped support for Python 3.7 so it can support numpy 1.23+

### CODE REVIEW

**Review by qlzh727 — APPROVED** [review]
Please wait for the final approval from Jonathan.

**Review by chenmoneygithub — APPROVED** [review]
Thank you!

File: setup.py
Comment (mattdangerw) [inline setup.py:39]: Hmm, not sure about this? Maybe let's leave this off for now.

This would mandate a newer version of numpy than colab has by default right now. And if people did have a reason to use 1.24, we are prohibiting them for what appears to be only a test breakage.

We've generally try to go strict for requirements, loose for setup.py, as describer here. https://stackoverflow.com/a/33685899

Comment (sampathweb) [reply setup.py]: Done.  Left setup.py as-is for numpy version.

**Review by mattdangerw — APPROVED** [review]
Thanks! Looks great

**Pull request merged**
